lib/resty/dns/resolver.lua

```diff
@@ -832,6 +839,69 @@ end
 
 
 function _M.query(self, qname, opts, tries)
+    if self.doh then
+        return _M.doh_query(self,qname,opts,tries)
```

Comment:
We should avoid exposing unnecessary internal methods, and the use of local functions is enough here

Reply:
done